tests/ttnn/unit_tests/test_multi_device.py

```diff
@@ -242,7 +242,7 @@ def test_multi_device_data_parallel_matmul_op(device_mesh):
     """Multidevice API: Data Parallel on matmul"""
     from ttnn import ShardTensorToMesh, ConcatMeshToTensor, ReplicateTensorToMesh
 
-    torch_input_a_tensor = torch.rand((4, 1, 32, 32 * device_mesh.get_num_devices()), dtype=torch.bfloat16)
+    torch_input_a_tensor = torch.rand((8, 1, 32, 32 * device_mesh.get_num_devices()), dtype=torch.bfloat16)
```

Comment:
can you change the batch to `device_mesh.get_num_devices()`? Also, just change the column dim because we don't need that to be a function of devices